examples/resources/powerstore_host/host_import.tf

```diff
@@ -0,0 +1,10 @@
+# Below are the steps to import host :
+# Step 1 - To import a host , we need the id of that host
+# Step 2 - To check the id of the host we can make Get request to host endpoint. eg. https://10.0.0.1/api/rest/host which will return list of all host ids.
```

Comment:
Is there any plan for creating a datasource for host ?? Just curious to know this

Reply:
yes in upcoming sprint datasource for host will be created.